docs/_docs/plugins/generators.md

```diff
@@ -102,3 +98,7 @@ Generators are only required to implement one method:
   </tbody>
 </table>
 </div>
+
+If your generator is contained within a single file, it can be named whatever you want but it should have an `.rb` extension. If your generator is split across multiple files, it should be packaged as a Rubygem to be published at https://rubygems.org/. In this case, the name of the gem depends on the availability of teh name at that site because no two gems can have the same name.
```

Comment:
This typo made me chuckle a bit, but we should probably fix it up. 😄 
```suggestion
If your generator is contained within a single file, it can be named whatever you want but it should have an `.rb` extension. If your generator is split across multiple files, it should be packaged as a Rubygem to be published at https://rubygems.org/. In this case, the name of the gem depends on the availability of the name at that site because no two gems can have the same name.
```